docs/src/main/tut/configurable-converters.md

```diff
@@ -1,6 +1,6 @@
 ## Configurable converters
 
-For some types, PureConfig cannot automatically derive a converter because there are multiple ways to convert a configuration value to them. For instance, for [`LocalDate`](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) PureConfig cannot derive a converter because there are multiple [`DateTimeFormatter`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html)s that can be used to convert a String into a `LocalDate`. Examples of different formats are `yyyy-mm-dd`, e.g. `"2016-01-01"`, and `yyyymmdd`, e.g. `"20160101"`. 
+For some types, PureConfig cannot automatically derive a converter because there are multiple ways to convert a configuration value to them. For instance, for [`LocalDate`](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) PureConfig cannot derive a converter because there are multiple [`DateTimeFormatter`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html)s that can be used to convert a String into a `LocalDate`. Examples of different formats are `yyyy-mm-dd`, e.g. `"2016-01-01"`, and `yyyymmdd`, e.g. `"20160101"`.
```

Comment:
I don't really know, I can't see it either... Maybe some whitespace? I guess I opened the file in IntelliJ and it reformatted the file in some way.